ecosystem/sep-0038.md

```diff
@@ -0,0 +1,176 @@
+## Preamble
+
+```
+SEP: 0038
+Title: Request for Quote API
+Author: Jake Urban <@jakeurban>
+Track: Standard
+Status: Draft
+Created: 2021-04-09
+Discussion: https://github.com/stellar/stellar-protocol/issues/901
+Version 1.0.0
+```
+
+## Summary
+
+This protocol enables anchors to accept off-chain assets in exchange for different on-chain assets, and vice versa. Specifically, it enables anchors to provide quotes that can referenced within the context of existing Stellar Ecosystem Proposals. How the exchange of assets is facilitated is outside the scope of this document.
+
+## Motivation
+
+Anchoring an asset and issuing an asset are distinct functions that have different buisness and technical requirements. However, issuing an asset has often been a prerequisite for anchoring an asset. This protocol enables anchors to transfer value on and off the Stellar network for their clients regardless of whether a one-for-one reserve-backed Stellar asset existed for the off-chain asset held by the anchor's clients.
+
+Removing this requirement for anchors also provides downstream benefits to ecosystem participants generally. Enabling anchors to accept any Stellar asset will naturally decrease liquidity fragmentation on the decentralized exchange, leading to greater market depth and tigher spreads between trading pairs.
+
+## Specification
+
+### Authentication
+
+All endpoints require authentication in the form of a [SEP-10](sep-0010.md) JSON Web Token (JWT) in the `Authorization` header of the request. Authenticated accounts must exist on the network in order to make requests to `POST /quote`, as described in the [Protecting Against Bad Actors](#protecting-against-bad-actors) section.
```

Comment:
❗ We should provide an example of how to specify the token in the `Authorization` header. I assume it would be as:

```
Authorization: Bearer <jwt>
```

I realize in some other SEPs we have that have a similar section to this don't provide an example, but they should, because there are multiple forms to specify an authorization header value.